dali/pipeline/operators/reader/loader/loader_test.cc

```diff
@@ -34,18 +34,16 @@ class DataLoadStoreTest : public DALITest {
 };
 
 typedef ::testing::Types<CPUBackend> TestTypes;
-string loader_test_image_folder = "/data/dali/benchmark";  // NOLINT
+string loader_test_image_folder = testing::dali_extra_path() + "/db/single/jpeg";  // NOLINT
 
 TYPED_TEST_SUITE(DataLoadStoreTest, TestTypes);
 
-const char* path = std::getenv("DALI_TEST_CAFFE_LMDB_PATH");
```

Comment:
Remove this from `https://github.com/NVIDIA/DALI/blob/1f63dfa105112b5ab34d8ccb8be88d90020da7dc/qa/L0_self-test/test.sh#L23` as well.

Reply:
And update https://github.com/NVIDIA/DALI/blob/master/dali/benchmark/caffe_alexnet_bench.cc#L43 as well.

Reply:
I don't won't to change the benchmark data, those are not exactly tests, but are meant to measure perf on the original dataset.

Reply:
Updated the L0.